src/main/java/gov/usgs/wma/waterdata/groundwater/S3Bucket.java

```diff
@@ -38,10 +37,13 @@ public String getKeyName() {
 	@Override
 	public void close() throws Exception {
 		try {
-			writer.close();
-			sendS3();
+
+			if (writer != null) {
+				writer.close();
+				sendS3();
+			} //else there is no content
 		} finally {
-			if (isDisposeFile()) {
+			if (file != null) {
```

Comment:
This shouldn't happen, but does happen in mock testing.  It seems reasonable to to just check.